astropy/tests/tests/test_runner.py

```diff
@@ -1,9 +1,11 @@
-from astropy.tests.runner import TestRunner, TestRunnerBase, keyword
```

Comment:
Ah, so it goes just by name. Renaming or importing locally make sense. I'd probably rename to `_TestRunner` but that's nitpicking.